docs/marko-testing-library/api.mdx

```diff
@@ -44,9 +44,9 @@ import {render, screen} from '@marko/testing-library'
 import Greeting from './greeting.marko'
 
 test('renders a message', async () => {
-  const {container} = await render(Greeting, {name: 'Marko'})
+  const {asFragment} = await render(Greeting, {name: 'Marko'})
```

Comment:
I'm pretty sure that `marko-testing-library` doesn't have an `asFragment` method so this change is probably only relevant to `react-testing-library` :)